src/map/battle.cpp

```diff
@@ -10517,6 +10525,19 @@ enum damage_lv battle_weapon_attack(struct block_list* src, struct block_list* t
 		}
 	}
 
+	// Poison React counter activates on normal poison attacks as well as attacks from poison-element enemies
```

Comment:
```suggestion
	// Poison React counter activates on attacks from poison-element enemies as well as normal poison attacks
```
Makes it easier to follow your IF logic, if it is in the same order...